gnmi_server/server_test.go

```diff
@@ -3487,6 +3487,77 @@ func TestClientConnections(t *testing.T) {
 	}
 }
 
+func TestNonExistentTableNoError(t *testing.T) {
```

Comment:
>TestNonExistentTableNoError [](http://example.com/codeflow?start=5&length=27)

Is there a test case for existing table no error?

Reply:
Added test